coronaqr.go

```diff
@@ -68,26 +121,16 @@ type VaccineRecord struct {
 }
 
 type TestRecord struct {
-	Target string `cbor:"tg" json:"tg"`
-	// "tt": {
-	//   "description": "Type of Test",
-	//   "$ref": "https://id.uvci.eu/DCC.ValueSets.schema.json#/$defs/test-type"
-	// },
+	Target DiseaseTargeted `cbor:"tg" json:"tg"`
```

Comment:
The formatting makes me think that you have not formatted this code using gofmt? https://blog.golang.org/gofmt

Reply:
Done. Thank you for teaching me :-)